build-system/compile/post-closure-babel.js

```diff
@@ -35,21 +35,6 @@ function loadSourceMap(file) {
   return null;
 }
 
-/**
- * @param {string} map
- * @return {function(string)}
- */
-function returnMapFirst(map) {
-  let first = true;
-  return function (file) {
-    if (first) {
-      first = false;
-      return map;
-    }
-    return loadSourceMap(file);
```

Comment:
Travis is failing a check for an unused reference since this was removed.

Looks like the method `loadSourceMap` can safely be removed now.